src/dataDisplay/chip/chip.d.ts

```diff
@@ -4,14 +4,15 @@ export interface ChipPropTypes {
     children?: React.ReactNode;
     className?: string;
     color?: 'active' | 'action' | 'alert' |
-        'alternate' | 'bright' | 'condition' |
-        'configuration' | 'disable' | 'highlight' |
-        'inverse' | 'inverse-alternate' | 'light' |
-        'nest' | 'outline' | 'primary' |
-        'secondary' | 'static' | 'subject' |
-        'success' | 'transparent' | 'warning',
+    'alternate' | 'bright' | 'condition' |
+    'configuration' | 'disable' | 'highlight' |
+    'inverse' | 'inverse-alternate' | 'light' |
+    'nest' | 'outline' | 'primary' |
+    'secondary' | 'static' | 'subject' |
+    'success' | 'transparent' | 'warning',
```

Comment:
Was ESLINT complaining?  I like 'em better indented over.
NAB of course! =)

Reply:
yep, ESLINT :)

Reply:
I'm gonna probably suppress it.  =)
I don't know why -- it seems reasonable to me to have that indented over when it's a continuation broken out into separate lines.  Like the rule seems a bit whack, or is perhaps misconfigured, IMHO.